src/nuclio/common/screens/create-function/function-from-template/function-from-template.component.js

```diff
@@ -436,6 +438,26 @@
                                                                                lodash.first(ctrl.projectsList);
         }
 
+        /**
+         * Sets class `last-line` to elements from the last row of the templates list.
+         */
+        function setLastLineClass() {
+            var TEMPLATE_WIDTH = 368;
+            var templates = $element.find('.function-template-wrapper');
+            var templatesWrapper = $element.find('.templates-wrapper');
+            var elementsPerLine = Math.floor(parseInt(templatesWrapper.css('width')) / TEMPLATE_WIDTH);
+            var countLastLineElements = lodash.size(templates) % elementsPerLine || elementsPerLine;
+            var lastLineElements = lodash.takeRight(templates, countLastLineElements);
+
+            lodash.forEach(templates, function (tmp) {
+                angular.element(tmp).removeClass('last-line');
+            });
+
+            lodash.forEach(lastLineElements, function (tmp) {
+                angular.element(tmp).addClass('last-line');
+            });
```

Comment:
As far as I know jQuery, you can call a method on the collection of elements that is returned from other methods.
Try this:
```js
templates.removeClass('last-line');
lastLineElements.addClass('last-line');
```